src/chains/ethereum/ethereum/src/forking/handlers/ws-handler.ts

```diff
@@ -20,6 +20,12 @@ export class WsHandler extends BaseHandler implements Handler {
     }>
   >();
 
+  // retry configuration
+  private retryIntervalBase: number = 2;
+  private retryCounter: number = 3;
+  private initialRetryCounter = this.retryCounter;
```

Comment:
What's the advantage of using `this` on the RHS, vs the same value already in scope?

Reply:
do you mean instead of using retry counter why not directly use 3?

Reply:
Thanks! It took a minute to reason about `this` outside the constructor. They'll be relocated into the constructor during transpilation.